src/app/src/components/LoginForm.jsx

```diff
@@ -106,7 +106,7 @@ class LoginForm extends Component {
                         />
                     </div>
                     <SendResetPasswordEmailForm />
-                    <ShowOnly when={error && error.length}>
+                    <ShowOnly when={!!(error && error.length)}>
```

Comment:
Interesting. Why did we have to do this? I could see myself forgetting to do this and expecting truthy values to just work.

Reply:
I set `when` as a `bool` prop-type in the `ShowOnly` component, but  I guess it doesn't automatically cast so it was complaining about receiving a number when there were errors. I noticed that the prior uses of `ShowOnly` had `length > 0` type expressions in them, and I guess this might be why.